src/AccessibilityInsights.SharedUx/Controls/TestTabs/TabStopControl.xaml.cs

```diff
@@ -363,9 +359,9 @@ private void StartRecordingEvent()
                 TabStopCount = 0;
                 IsTabStopLooped = false;
 
-                this.EventHandler?.RegisterAutomationEventListener(EventType.UIA_AutomationFocusChangedEventId, this.OnFocusChanged);
-
-                this.lvElements.Items.Clear();
+                CurrentElement = null;
```

Comment:
`CurrentElement = null;` should already have been happening somewhere. Perhaps we lost it inadvertently.